e2e/cypress/integration/integrations/outgoing_webhook/delete_outgoing_webhook_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+let testTeam;
+
+describe('Integrations', () => {
+    before(() => {
+        // # Create test team, channel, and webhook
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Visit the webhook page
+            cy.visit(`/${team.name}/integrations/outgoing_webhooks/add`);
+        });
+    });
+
+    it('MM-T617 Delete outgoing webhook', () => {
+        // # Create an outgoing webhook with trigger word "testing"
+        cy.get('#displayName', {timeout: TIMEOUTS.ONE_MIN}).type('New Outgoing Webhook');
+        cy.get('#description').type('Description');
+        cy.get('#channelSelect').select('Town Square');
```

Comment:
Is there a way to select the created channel from the drop down? I tried several variants but could not figure out how to get the proper channel so opted for TS.

Reply:
In your `before` you can have something like,
```
        cy.apiInitSetup().then(({team, channel}) => {
            testTeam = team;
            testChannel = channel;
        });
```
then you select channel you've created like,
```
cy.get('#channelSelect').select(testChannel.display_name);
```

Reply:
Now working with cy.apiCreateWebhook to set up.